substrate-archive-common/src/util.rs

```diff
@@ -43,7 +43,7 @@ pub fn init_logger(std: log::LevelFilter, file: log::LevelFilter) -> Result<(),
 		.level_for("staking", log::LevelFilter::Warn)
 		.level_for("cranelift_codegen", log::LevelFilter::Warn)
 		.level_for("header", log::LevelFilter::Warn)
-		.level_for("", log::LevelFilter::Error)
+		// .level_for("", log::LevelFilter::Error)
```

Comment:
We don't want errors logged?

Reply:
Ah, this just filters out the target for fern. During my testing I was wondering if fern was maybe culling some of the state tracing logs, this line just tells fern to ignore any log with target "" and is above level Error